twisted/protocols/memcache.py

```diff
@@ -299,20 +305,22 @@ def cmd_ERROR(self):
         cmd.fail(NoSuchCommand())
 
 
-    def cmd_CLIENT_ERROR(self, errText):
+    def cmd_CLIENT_ERROR(self, errBytes):
```

Comment:
I didn’t notice it was public – whoops! In that case, the bytes still have to represent text, so maybe it would be best to just keep the name as-is.